arcgis-ios-sdk-samples/Layers/ArcGIS vector tiled layer (custom style)/VectorTileCustomStyleViewController.swift

```diff
@@ -17,40 +17,122 @@ import UIKit
-        let centerPoint = AGSPoint(x: 1990591.559979, y: 794036.007991, spatialReference: .webMercator())
-        map.initialViewpoint = AGSViewpoint(center: centerPoint, scale: 88659253.829259947)
-        
-        //assign map to map view
-        mapView.map = map
-        
-        //show the default vector tiled layer
-        showSelectedItem(itemIDs.first!)
+    func loadVectorTiledLayer(url vectorTiledLayerURL: URL, itemID: String) {
+        // Get a temporary URL to download the custom style.
+        let temporaryURL = getTemporaryURL()
+        // Create a vector tile cache using the local vector tile package.
+        let vectorTileCache = AGSVectorTileCache(name: "dodge_city")
+        // Create a portal item with the URL.
+        let portalItem = AGSPortalItem(url: vectorTiledLayerURL)!
+        // Create a task to export the custom style resources.
+        let task = AGSExportVectorTilesTask(portalItem: portalItem)
+        // Get the AGSExportVectorTilesJob.
+        exportVectorTilesJob = task.exportStyleResourceCacheJob(withDownloadDirectory: temporaryURL)
+        // Start the job.
+        exportVectorTilesJob?.start(
+            statusHandler: { (status: AGSJobStatus) in
+                SVProgressHUD.show(withStatus: status.statusString())
+            }, completion: { [weak self] (result, error) in
+                SVProgressHUD.dismiss()
+                guard let self = self, let map = self.mapView.map else { return }
+            if let result = result {
+                // Get the item resource cache from the result.
+                let itemresourceCahce = result.itemResourceCache
+                switch itemID {
+                case "e01262ef2a4f4d91897d9bbd3a9b1075":
+                    // Create a vector tiled layer with the vector tiled cache and the item resource cache.
+                    let dayVectorTiledLayer = AGSArcGISVectorTiledLayer(vectorTileCache: vectorTileCache, itemResourceCache: itemresourceCahce)
+                    map.basemap = AGSBasemap(baseLayer: dayVectorTiledLayer)
+                    self.dayVectorTiledLayer = dayVectorTiledLayer
+                case "ce8a34e5d4ca4fa193a097511daa8855":
+                    // Create a vector tiled layer with the vector tiled cache and the item resource cache.
+                    let nightVectorTiledLayer = AGSArcGISVectorTiledLayer(vectorTileCache: vectorTileCache, itemResourceCache: itemresourceCahce)
+                    map.basemap = AGSBasemap(baseLayer: nightVectorTiledLayer)
+                    self.nightVectorTiledLayer = nightVectorTiledLayer
+                default:
+                    fatalError("Unidentified item ID")
+                }
+            } else if let error = error {
+                // Handle errors.
+                self.presentAlert(error: error)
+            }
+            }
+        )
+        try? FileManager.default.removeItem(at: temporaryURL)
     }
     
     private func showSelectedItem(_ itemID: String) {
```

Comment:
By tapping on the custom style twice, it seems to disappear. See the screenshot below.
![select-bug](https://user-images.githubusercontent.com/9660181/97644755-6a468880-1a08-11eb-88ec-77d5ea7a38a8.gif)